lib/vagrant-openshift/action/install_origin_base_dependencies.rb

```diff
@@ -215,6 +215,12 @@ def call(env)
   sudo lvcreate -n docker-data -l 90%FREE /dev/${VG}
   sudo lvcreate -n docker-metadata -l 50%FREE /dev/${VG}
   sudo sed -i "s,^DOCKER_STORAGE_OPTIONS=.*,DOCKER_STORAGE_OPTIONS='-s devicemapper --storage-opt dm.datadev=/dev/${VG}/docker-data --storage-opt dm.metadatadev=/dev/${VG}/docker-metadata'," /etc/sysconfig/docker-storage
+
+  sudo lvcreate -n openshift-xfs-tmp -l 100%FREE /dev/${VG}
+  sudo mkfs.xfs /dev/${VG}/openshift-xfs-tmp
+  sudo mkdir -p /tmp/openshift
```

Comment:
Ah.  Sorry I forgot this directory was already being used.  I was thinking you were adding a new dir only used by specific tests.   As it is now, I don't think this dir is going to be big enough to do everything /tmp/openshift was doing.  Do you know big it gets today?